modules/ml/src/main/java/org/apache/ignite/ml/math/MatrixStorage.java

```diff
@@ -55,6 +55,13 @@
     public int storageMode();
 
     /**
+     * @return Matrix storage mode.
```

Comment:
looks like a typo, I would expect javadoc to say "access mode" instead of "storage mode" here